739 add exercise template

B1tF8er opened the pull request:
Solves #739

robkeim commented on docs/new_exercise_issue_template.md:
```diff
@@ -0,0 +1,10 @@
+There is a new exercise, [exercise-name](https://github.com/exercism/problem-specifications/blob/master/exercises/exercise-name/description.md), which data can be found here: https://github.com/exercism/problem-specifications/tree/master/exercises/exercise-name
```
IMO it would be nice to have `EXERCISE-NAME` all in caps here and throughout this file. That way you can easily find all of the instances that need to be replaced.

B1tF8er replied:
Got it, will make the changes

robkeim commented on docs/new_exercise_issue_template.md:
```diff
@@ -0,0 +1,10 @@
+
+* Create a directory for the new exercise in the `exercises` directory (note: this has to match the exact exercise name).
+* Add a new entry to the [config.json](https://github.com/exercism/csharp/blob/master/config.json) file.
+* Create a generator to automatically convert the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/rest-api/canonical-data.json) to a test file. For more information on how to do this, check the [generators docs](https://github.com/exercism/csharp/blob/master/docs/GENERATORS.md).
```
Replace `rest-api` with `EXERCISE-NAME`

B1tF8er replied:
Got it, will make the changes

B1tF8er commented on docs/new_exercise_issue_template.md:
```diff
@@ -0,0 +1,10 @@
+* Create a generator to automatically convert the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/EXERCISE-NAME/canonical-data.json) to a test file. For more information on how to do this, check the [generators docs](https://github.com/exercism/csharp/blob/master/docs/GENERATORS.md).
+* Create a stub implementation file that contains the code necessary to compile (which are the methods called by the test file), but lacks an implementation. (see [this example](https://github.com/exercism/csharp/blob/master/exercises/two-fer/TwoFer.cs)).
+* Create an implementation file that passes all the tests.
+* Create a project file for the project (you can probably best copy and modify one of the existing projects).
```
ok, I will make the changes

B1tF8er commented:
I moved new_exercise issue template to `./github/ISSUE_Template` and renamed it to `new_exercise.md`, so if this PR is approved you will be able to use the template using this querystring in the URI
* https://github.com/exercism/csharp/issues/new_**?template=new_exercise.md**_

ErikSchierboom commented:
Great work @B1tF8er! Let's wait for @robkeim to finish his review, and then this is ready to merge!

robkeim commented:
Thanks a lot for your contribution @B1tF8er! 🎉

B1tF8er commented:
Glad I could help

ErikSchierboom commented:
Thanks a lot!
